liblouis/lou_translateString.c

```diff
@@ -2922,10 +2925,10 @@ resolveEmphasisPassages(EmphasisInfo *buffer, const EmphasisClass *class,
 			in_word = 1;
 			last_word_start = i;
 		} else /* check if at end of word */
-			if (in_word && !(wordBuffer[i] & WORD_CHAR)) {
-				in_word = 0;
-				last_word_end = i;
-			}
+				if (in_word && !(wordBuffer[i] & WORD_CHAR)) {
+			in_word = 0;
+			last_word_end = i;
+		}
```

Comment:
Seems like a unmotivated whitespace change?

Reply:
I think this might be due to using a different version of clang-format. Can you try to use version 14?

Reply:
I use the LLVM toolchain of version 12. This minor formatting issue is caused by `make format-sources`, and could be fixed by adding a curly brace for this single-statement branch.

Reply:
I use the LLVM toolchain of version 12. This minor formatting issue is caused by `make format-sources`, and could be fixed by adding curly braces for this single-statement branch.